css/styles.css

```diff
@@ -278,3 +282,104 @@ footer > .inner {
     }
 }
 
+@media (max-width: 760px) {
+    .wrapper > *,
+    header > *,
+    footer > * {
+        max-width: 690px;
+    }
+
+    .footer-menu {
+        top: 30px;
+        left: 20px;
+        right: 20px;
+        flex-wrap: wrap;
+        justify-content: center;
+    }
+
+    .footer-menu >*:not(:last-child) {
+        margin-right: 20px;
+        margin-bottom: 10px;
+    }
+}
+
+@media (max-width: 599px) {
+    .wrapper > *,
+    header > *,
+    footer > * {
+        max-width: 530px;
+    }
+
+    header > .inner {
+        height: 170px;
+    }
+
+    .logo {
+        width: 100px;
+        height: 120px;
+    }
+
+    .some-card {
+        flex: 0 1 100%;
+        margin-bottom: 20px;
+    }
+
+    .phone {
+        width: 200px;
+    }
+    .telegram {
+        width: 160px;
+    }
+}
+
+@media (max-width: 477px) {
+    .footer-menu {
+        top: 40px;
+        flex-direction: column;
+    }
+
+    footer > .inner {
+        height: 300px;
+    }
+
+    .phone {
+        width: 160px;
+        height: 20px;
+        top: 90px;
+        transform: translate(0%, 0);
+        left: auto;
+        right: 20px;
+    }
+
+    .telegram {
+        width: 110px;
+        height: 22px;
+        top: 130px;
+        transform: translate(0%, 0);
+        left: auto;
+        right: 20px;
+    }
+}
+
+@media (max-width: 320px) {
```

Comment:
Add to the selector `html, body` new rule `min-width: 320px`. And remove this media query.